code/controllers/subsystem/ticker.dm

```diff
@@ -56,13 +56,15 @@ var/datum/subsystem/ticker/ticker
 
 	var/news_report
 
+	var/list/prioritized_jobs = list()
```

Comment:
SSjob in [here](https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/jobs.dm)